src/components/Chip/Chip.scss

```diff
@@ -40,13 +71,28 @@ $vertical-padding: ($min-height - line-height(body) - rem(2px)) / 2;
     margin: 0 ($height / 4) 0 (-1 * $horizontal-padding);
     height: $height;
     width: $height;
-    border-radius: 50%;    
+    border-radius: 50%; 
+    display: inline-block;  
+    vertical-align: middle;
 }
 
-.Remove {
-    cursor: pointer;
-    flex-shrink: 0;
-    margin: ($horizontal-padding / 2) (-1 * $horizontal-padding / 2) ($horizontal-padding / 2) ($horizontal-padding / 2);
+body .Remove {
```

Comment:
@Farhan-emgage why was body added to this selector? Also, color's should be expressed using the color() mixin and margin/padding values typically use variables to allow for consistency and easy adjustment.

Reply:
@weegrom  Actually previously remove icon was in link. but now we took it in button. For making remove button alignment with sibling button we have to add some padding. but in chip component make changes for button style there are two ways either we have to place important or we apply strong relation. so i apply body before remove class.